internal/report/testdata/sample/sample.go

```diff
@@ -0,0 +1,41 @@
+// sample program that is used to produce some of the files in
+// pprof/internal/report/testdata.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"runtime/pprof"
+)
+
+var cpuProfile = flag.String("cpuprofile", "", "where to write cpu profile")
+
+func main() {
+	flag.Parse()
+	f, err := os.Create(*cpuProfile)
+	if err != nil {
+		log.Fatal("could not create CPU profile: ", err)
+	}
+	if err := pprof.StartCPUProfile(f); err != nil {
+		log.Fatal("could not start CPU profile: ", err)
+	}
+	defer pprof.StopCPUProfile()
+	busyLoop()
+}
+
+func busyLoop() {
```

Comment:
FWIW, I am not sure Golang includes the DWARF inline debug information, at least I don't see the inline expansion working with this test case - https://aalexand.github.io/scratch/busyloop.html. We can improve it later I guess.

Reply:
Agree, it's better to avoid fakes as much as possible.